sipa/templates/base.html

```diff
@@ -357,6 +357,13 @@ <h2>{{ _("Kontakt") }} </h2>
                                 <hr>
 
                                 <div class="row">
+                                    <div class="col-xs-4 text-center">
+                                        <a class="btn btn-social-icon btn-mastodon"
+                                           href="https://social.anoxinon.de/@ag_dsn"
+                                           aria-label="Mastodon">
+                                            <span class="fa fa-mastodon" aria-hidden="true"></span>
```

Comment:
Icon doesn't exist in fontawesome 4.
I upgraded our FA version to 6 with my latest commit and fixed the icon.